schema/crdb/11.0.0/up2.sql

```diff
@@ -0,0 +1,2 @@
+CREATE UNIQUE INDEX IF NOT EXISTS root_of_trust_page_properties
```

Comment:
This does make me slightly nervous (because the values are so big and the domain of possible values is so big) but I guess it's not really worse than any other string and in practice we don't expect to have that many values here.

Reply:
I'm also slightly nervous about this, but (a) I believe these change even less frequently than cabooses (e.g., when keys are revoked, for which we don't yet have a process) and (b) I'm not sure what an alternative would be. Without this index we wouldn't be able to ensure uniqueness "by hand", right, since querying to see if a given page exists would require a full table scan?

Reply:
Agreed.